views/layouts/main.html

```diff
@@ -25,15 +25,15 @@
 	<script src="https://polyfill.io/v2/polyfill.min.js?features=fetch,default"></script>
 </head>
 <body class="o-registry-ui">
-	<div class="beta-notice">The new Origami Registry is currently in <strong>beta</strong>,
-we <a href="mailto:origami.support@ft.com?subject=New Origami Registry">appreciate feedback</a>!
-<a href="http://registry.origami.ft.com{{requestUrl}}">Click here to get back to the safety of the live registry</a>.</div>
+
+	{{> beta-notice}}
 
 	{{> header/main }}
 
 	<main role="main">
 		{{{ body }}}
 	</main>
 
+	{{> footer}}
```

Comment:
The footer's looking a little odd when running locally:

<img width="1280" alt="screen shot 2018-02-26 at 09 42 06" src="https://user-images.githubusercontent.com/138944/36663945-1b5c0ee0-1adb-11e8-86d3-7029aa722abd.png">